contrib_bots/lib/DefineBot.py

```diff
@@ -0,0 +1,77 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+import logging
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the user inputs. It
+    looks for messages starting with '@define'.
+    '''
+
+    def __init__(self):
+        self._DEFINITION_API_URL = 'https://owlbot.info/api/v1/dictionary/{}?format=json'
```

Comment:
I think it would be better if these are class variables instead of instance variables since they don't change.